.gitignore

```diff
@@ -6,8 +6,6 @@
 .idea/**/shelf
 .idea/compiler.xml
 .idea/encodings.xml
-.idea/misc.xml
```

Comment:
@brian-mcnamara Is it ok to check in `misc.xml`?

Reply:
Yep, misc is fine.